README.md

```diff
@@ -5,7 +5,11 @@
 [![Build Status][travis-image]][travis-url]
 [![Test Coverage][coveralls-image]][coveralls-url]
 
-Merge objects using descriptors.
+Merge source object "own" properties onto a destination object, including 
+non-enumerable "own" properties, and preserve each property's descriptors. 
+Descriptors are configuration data associated to each property such as whether 
+the property is `writable`, `enumerable`, and/or `configurable`.  This data 
```

Comment:
So getters, setters, and the value is also copied over, which isn't listed. This (at least to me?) reads almost like the value of the property is not actually copied over. WDYT?

Also there is still a double space on this line by accident.

Reply:
Your right.  I could add a little more to clarify that.  Give me a moment.